README.md

```diff
@@ -62,22 +62,25 @@ might want to use `Cluster List` or `Workload List` commands. Finally, you can
 cleanup with a `Cluster Delete`.
 
 ## Cluster Create
-*   Cluster Create (provision on-demand capacity):
```

Comment:
Wait is this the CR you meant to put here? Doesn't seem to have anything to do with the title:
```
Xpk fix parallel node pool create #27
```

Reply:
I build this change on top of the capacity arguments one purposefully for testing the two features together lol. Merged the capacity changes in and this PR looks clean now.